pkg/proxy/proxy_test.go

```diff
@@ -35,10 +35,20 @@ func newFakeProxy(existingTillerReleases []AppOverview) *Proxy {
 		status := release.Status_DEPLOYED
 		if r.Status == "DELETED" {
 			status = release.Status_DELETED
+		} else if r.Status == "FAILED" {
+			status = release.Status_FAILED
+		}
+		version := int32(1)
+		// Increment version number if the same release name has been already added
```

Comment:
Ohhh, now I worked out that this is the Helm _revision_ version, not the chart version. My confusion was because I thought you were randomly bumping the chart version even though we provide that in the AppOverview. Can you edit the comment here to point out that this is the Helm release version/revision?